src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -186,7 +186,7 @@ private async Task<SearchResult> SearchImpl(SearchOptions searchOptions, bool hi
                                                .AcceptVisitor(IncludeRewriter.Instance)
                                            ?? SqlRootExpression.WithDenormalizedExpressions();
 
-            using (SqlConnectionWrapper sqlConnectionWrapper = _sqlConnectionWrapperFactory.ObtainSqlConnectionWrapper(true))
+            using (SqlConnectionWrapper sqlConnectionWrapper = await _sqlConnectionWrapperFactory.ObtainSqlConnectionWrapperAsync(cancellationToken, true))
```

Comment:
Changing calls to `SqlConnectionWrapperFactory` to be async (see PR [#102](https://github.com/microsoft/healthcare-shared-components/pull/102)).